app/controllers/account/comments_controller.rb

```diff
@@ -0,0 +1,36 @@
+class Account::CommentsController < ApplicationController
+  before_action :authenticate_user!
+
+  def create
+    @comment = current_user.comments.new(comment_params)
+    if @comment.save
+      respond_to_format
+    else
+      respond_to_format(render(js: "alert('Comment is invalid!');"))
+    end
+  end
+
+  def destroy
+    @comment = resource
+    if current_user.privileged?
+      @comment.destroy
+      respond_to_format
+    else
+      respond_to_format(render(js: "alert('No rights to delete this comment');"))
+    end
+  end
+
+  private
+
+  def respond_to_format(block = nil)
+    respond_to { |format| format.js { block } }
+  end
+
+  def resource
+    Comment.find(params[:id])
```

Comment:
as an admin or one tenant I can delete any comment in system, by changing ID in URL.

use current_tenant.

maybe add column tenant_id to comments